apm-sniffer/apm-sdk-plugin/dubbo-3-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo3/DubboInstrumentation.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.plugin.asf.dubbo3;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.ConstructorInterceptPoint;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.InstanceMethodsInterceptPoint;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.ClassInstanceMethodsEnhancePluginDefine;
+import org.apache.skywalking.apm.agent.core.plugin.match.ClassMatch;
+import org.apache.skywalking.apm.agent.core.plugin.match.NameMatch;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+public class DubboInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
+
+    private static final String ENHANCE_CLASS = "org.apache.dubbo.monitor.support.MonitorFilter";
```

Comment:
And this is targeting the same class of Dubbo 2. What do I miss here? I thought Dubbo v3 is different from Dubbo v2?

Reply:
I'm new to skywalking, I didn't dig for a deeper understanding of skywalking. When I finished current state of this pr, only tested using my dubbo3 demo, and it works.
As you pointed out, I believe it won't work as expect when using dubbo 2.7.
Converted to draft.

Reply:
> As you pointed out, I believe it won't work as expect when using dubbo 2.7.

We have a plugin for Dubbo 2.7, my question is what are the differences between these two. They seems very closing to each other, especially targeting same class(name)?

Reply:
Differences: (not all, ones currently I know)
1. RpcContext.getContext() didn't work;
2. attachment divided into `ClientAttachment` and ServerAttachment;
3. dubbo constants definition class divided into multiple classes (like `org.apache.dubbo.common.constants.CommonConstants`);
4. methods added in `org.apache.dubbo.common.URL`

Reply:
OK then, follow https://github.com/apache/skywalking-java/pull/73#discussion_r754864763, You should be good.